infra/feast-operator/internal/controller/featurestore_controller_pvc_test.go

```diff
@@ -67,6 +67,8 @@ var _ = Describe("FeatureStore Controller-Ephemeral services", func() {
 		onlineStoreMountPath := "/online"
 		registryMountPath := "/registry"
 
+		storageClassName := "default"
```

Comment:
it's a bit confusing to use "default" as the `storageClassName` here, since you're testing a scenario where a user actually defines the storage class name... as opposed to leaving it blank and using the default. maybe use a different name? "test" ?